azure-pipelines.yml

```diff
@@ -28,6 +28,29 @@ jobs:
       DOCKERHUB_PASSWORD: $(DockerPassword)
       GCP_SERVICE_ACCOUNT_KEY: $(GcpServiceAccountKey)
 
+- job: build_container_windows
+  dependsOn: []
+  pool:
+    vmImage: 'windows-latest'
+  steps:
+  - bash: |
+      curl -fsSL --output "$TEMP/gcloud-sdk-installer.exe" "$GCLOUD_SDK_INSTALLER_URL"
+      echo "$GCLOUD_SDK_INSTALLER_CHECKSUM $TEMP/gcloud-sdk-installer.exe" | sha256sum --check
+      export MSYS2_ARG_CONV_EXCL=*
+      $TEMP/gcloud-sdk-installer.exe /S /noreporting /nostartmenu /nodesktop
+      echo "##vso[task.prependpath]C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\bin"
+    env:
+      GCLOUD_SDK_INSTALLER_URL: https://dl.google.com/dl/cloudsdk/channels/rapid/GoogleCloudSDKInstaller.exe
+      GCLOUD_SDK_INSTALLER_CHECKSUM: 307e3d883dff23d4de739555696c5d2688c5612e2aa3361cd31b85e2ac2e7de3
```

Comment:
I think this will break when google update cloud-sdk? no?

Reply:
ah yeah, they don't seem to provide a versioned URL for that installer but I can dig around